bin/start

```diff
@@ -34,7 +34,9 @@ EOF
 }
 
 runScriptWithSummon() {
-    summon --provider summon-conjur --environment $SUMMON_ENV -f ./summon/secrets.yml $1
+    # CONJUR_ACCOUNT is set in bootstrap.env to a test value, which isn't valid for
```

Comment:
so should we remove it from the boostrap.env file?

Reply:
Looks like it might be needed for connecting to a temporary test conjur instance?